core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1125,6 +1132,46 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
   def getStorage(credential: Credential) =
     new Storage.Builder(httpTransport, jsonFactory, credential).setApplicationName(appName).build()
 
+  override def getBucketMetrics(
+    projectId: GoogleProjectId
+  ): BucketMetricsResponse = {
+    val metricServiceClient = MetricServiceClient.create(
+      MetricServiceSettings
+        .newBuilder()
+        .setCredentialsProvider(FixedCredentialsProvider.create(getBucketServiceAccountCredentials))
+        .build()
+    )
+    val projectName = projectId.value
+    val now = System.currentTimeMillis
+    val fiveMinutesAgo = now - 5 * 60 * 1000
```

Comment:
The metric is sampled every 300 seconds, so going back 5 minutes should give us the most recent value (although it's only measured once a day).